.travis.yml

```diff
@@ -5,18 +5,28 @@ language: java
 jdk:
   - oraclejdk8
 
-env:
-  - TRAVIS_NODE_VERSION="stable"
-  - TRAVIS_NODE_VERSION=8
-  - TRAVIS_NODE_VERSION=6
-
 branches:
   only:
     - "master"
 
-before_install:
-  - nvm install $TRAVIS_NODE_VERSION
 
-install: "npm install"
+jobs:
+  include:
+
+    - stage: "Build and test application"
```

Comment:
@catalan-adobe can you include this in https://github.com/vvo/selenium-standalone/pull/387 ? It would be extremely beneficial for future PRs addressing Docker

Reply:
I continued investigating the way to handle update of the Docker image and I don't think Travis is the right place as currently the project is released via `scripts/release.sh`.
I'm currently working on adding the Docker image update+deploy steps in the release script.